### PR TITLE
feat: add transaction count to /block.all and /block.get

### LINKED ISSUE #1231: Add transaction count to block.all and block.get
This is to implement block explorer design that shows the number of transactions for a block. 
We are currently doing this with a `SELECT` statement and would like to move the full functionality to corresponding API endpoints. 

See [here](https://gallery.io/projects/MCHbtQVoQ2HCZfx8djn6JiP_/files/MCEJu8Y2hyDScXV2LYjxdfr91NLrkIHlsQs) and [here](https://gallery.io/projects/MCHbtQVoQ2HCZfx8djn6JiP_/files/MCEJu8Y2hyDScf3vzax-9A3x4A7SaeBV8D0).

**Current behaviour**:

```
  {
      "blknum": 92000,
      "eth_height": 5364313,
      "hash": "0x87c39b60f49c18d5366d7c04b9de0d87d8b5a83dd2a4e03162a74e4aa449b483",
      "timestamp": 1572603821
    }
``` 

**Desired behaviour**:
```
 {
      "blknum": 92000,
      "eth_height": 5364313,
      "hash": "0x87c39b60f49c18d5366d7c04b9de0d87d8b5a83dd2a4e03162a74e4aa449b483",
      "timestamp": 1572603821, 
      "tx_count": 5
    }
```

### PR DESCRIPTION
**Pull request by okalouti**
Closes #1231 

## Overview

This PR is adds `tx_count` in the block object returned by `/block.all` and `/block.get.` The intended use case is for the block explorer to display transaction count per block.

## Changes

- Modifies queries in `omg_watcher_info/db/block.ex` to extract transaction count
- Adds tests in `test/omg_watcher_info/db/block.exs` for the above, and changes an existing test to use the `:phoenix_ecto_sandbox` fixture instead of `:initial_blocks`
- Updates Swagger specs

## Testing

Run `mix test test/omg_watcher_info/db/block_test.exs` for the recent changes 

Thanks @unnawut for guidance and thanks in advance for feedback.

### CODE REVIEW

**Comment by okalouti** [comment]
This is causing a test for `transaction.get` to fail. Its return object now includes `tx_count: nil`. 
Will think how best to handle this and revert.

File: apps/omg_watcher_info/lib/omg_watcher_info/db/transaction.ex [hunk 1]
```diff
@@ -51,7 +51,7 @@ defmodule OMG.WatcherInfo.DB.Transaction do
         __MODULE__,
         where: [txhash: ^hash],
         preload: [
-          :block,
+          block: ^DB.Block.base_query(),
```
Comment (unnawut) [inline apps/omg_watcher_info/lib/omg_watcher_info/db/transaction.ex]: Amazing composability 👏

File: apps/omg_watcher_info/test/omg_watcher_info/db/block_test.exs [hunk 2]
```diff
@@ -25,13 +25,48 @@ defmodule OMG.WatcherInfo.DB.BlockTest do
   @eth OMG.Eth.RootChain.eth_pseudo_address()
 
   describe "get/1" do
-    @tag fixtures: [:initial_blocks]
+    @tag fixtures: [:phoenix_ecto_sandbox]
```
Comment (unnawut) [inline apps/omg_watcher_info/test/omg_watcher_info/db/block_test.exs]: Thanks!

File: apps/omg_watcher_info/lib/omg_watcher_info/db/block.ex [hunk 3]
```diff
@@ -63,13 +66,22 @@ defmodule OMG.WatcherInfo.DB.Block do
...
       from(
-        __MODULE__,
+        block in base_query(),
         where: [blknum: ^blknum]
       )
 
     DB.Repo.one(query)
   end
 
+  def base_query() do
```
Comment (unnawut) [inline apps/omg_watcher_info/lib/omg_watcher_info/db/block.ex]: Can you add a test for this new function into `DB.BlockTest`?

Comment (okalouti) [reply apps/omg_watcher_info/lib/omg_watcher_info/db/block.ex]: Done de5e2c7

**Comment by unnawut** [comment]
The PR title will become the squash commit message hence the rename.

**Pull request merged**